### ISSUE TITLE
Custom configurations not supported

### ISSUE THREAD

**Issue by dineshpannu** [firegiant/HeatWaveSupport#12]
### HeatWave Version

0.9.0.5

### Visual Studio Version

17.3.6

### Repro Steps

1. Open a solution and add new project of type "MSI Package (WiX v4)"
2. On file menu click on Build -> Configuration Manager
3. Click on the Configuration drop down next to the installer project and select ```<New...>```
4. Fill name, copy setttings from ```<Empty>```, don't tick ```Create new solutions configurations```. Click OK.

### Actual Result

In the UI I found no new configuration was added. 

In the .wixproj, a single configuration is added to the end of <Configurations>. This configuration is replaced if a new configuration is created. If configurations are deleted or added manually in the .wixproj, using the UI will force "Debug" and "Release" to return.

When the $Configuration variable is used, it mirrors what the UI reports configuration as - either "Debug" or "Release".

### Expected Result

I expected new configurations created in the UI to exist in Configuration Manager and for $Configuration to be populated correctly. All of our builds have different installer outputs depending on Configuration and none of our solutions use Debug or Release configurations.

**Comment by FireGiantHelp** [comment]
We have a backlog item to support custom configurations in HeatWave.

**Comment by FireGiantHelp** [comment]
The dev team looked into this issue but do not have a solution yet. The issue remains on our backlog.

**Comment by PeterESoft** [comment]
If the dev team still has no solution I can offer the following solution in the file FireGiant.HeatWave.DesignTime.targets. For me it works to have default configurations and also to add custom configurations via Visual Studio Configuration Manager.

```
<!-- Declare the valid project configurations -->
<PropertyGroup>
  <Configurations Condition="'$(Configurations)' == ''">Debug;Release</Configurations>
  <Platforms Condition="'$(Platforms)' == ''">x86;x64;ARM64</Platforms>
</PropertyGroup>
```

**Comment by FireGiantHelp** [comment]
We have updated the HeatWave design-time targets with the same `Condition`'s you presented above and included those changes in the `0.9.5` release that shipped today.

Thank you for your feedback.
